nevergrad/benchmark/experiments.py

```diff
@@ -76,6 +76,7 @@ def yawidebbob(seed: tp.Optional[int] = None) -> tp.Iterator[Experiment]:
             instrum = ng.p.Tuple(*(ng.p.TransitionChoice(range(arity)) for _ in range(nv)))
             for discrete_func in [corefuncs.onemax, corefuncs.leadingones, corefuncs.jump]:
                 dfunc = ExperimentFunction(discrete_func, instrum)
+                dfunc._descriptors.update(arity=arity)
```

Comment:
@teytaud this is all that was necessary

Reply:
(the other changes are unrelated, because mypy was starting to complain for weird reasons)